helm/vmss-prototype/templates/vmss-prototype.yaml

```diff
@@ -13,30 +13,30 @@
 apiVersion: v1
 kind: ServiceAccount
 metadata:
-  name: kamino
+  name: kamino-{{ $jobName }}
   namespace: {{ .Release.Namespace }}
 ---
 apiVersion: rbac.authorization.k8s.io/v1
 kind: ClusterRole
 metadata:
-  name: node-updater
+  name: kamino-{{ $jobName }}
 rules:
 - apiGroups: [""]
-  resources: ["nodes"]
+  resources: ["nodes", "pods"]
```

Comment:
Drain needs "pods/eviction" in addition to "pods"

Reply:
done, thx!